docusaurus/docs/introduction/plugin-types-usage

```diff
@@ -0,0 +1,133 @@
+
+## Data source plugins
+
+In plugin development, you can create new data source plugins to use Grafana with new databases, similar to existing plugins such as [MongoDB](https://grafana.com/grafana/plugins/grafana-mongodb-datasource/) or [Google BigQuery](https://grafana.com/grafana/plugins/grafana-bigquery-datasource/).
+
+### Usage of data-source plugins
+
+Create and configure a Grafana data source when you want to work with data provided by a third-party service for use in Grafana Dashboards, Explore, Alerting, and so on.
+
+The data source API [(`/api/datasources`)](https://grafana.com/docs/grafana/latest/developers/http_api/data_source/) exposes endpoints to create, read, update, and delete data sources.
+
+Given any installed plugin of type `datasource`, you can create and configure any number of data sources per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations). After you create zero to infinity data sources, they are persisted in Grafana's database. 
+
+:::note
+
+To distinguish a Grafana data source from a data-source plugin, we sometimes refer to the latter as a _data-source instance_, that is, a configured Grafana data source with a `plugin id` type of `datasource`.
+
+:::
+
+
+### Provisioning of data sources
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) to configure your data source plugins.
+
+Data sources can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/) so that you can use YAML files when Grafana starts up or on demand. Note that a plugin of type `datasource` must be installed before it can be provisioned.
+
+### Bundling of dashboards
+
+Data-source plugins can include dashboards by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a data source plugin in the `General` folder when it is imported.
+
+## App plugins
+
+Development of app plugins, also known as apps or applications, allows you to bundle data sources and panels to provide a seamless monitoring experience, similar to existing plugins such as [Zabbix](https://grafana.com/grafana/plugins/alexanderzobnin-zabbix-app/).
+
+### Usage of apps
+
+You can configure a Grafana app when you want to integrate with a third-party service and optionally provide custom pages and such within Grafana. Given any installed plugin of type `app`, you can enable them once per per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations) and they are persisted in Grafana’s database. 
+
+Use the plugin settings API [`/api/plugins`](https://grafana.com/api/plugins), `/api/plugins/\<plugin id\>/settings/`) for endpoints to read and update app settings. For example, you can enable or disable the app with your preferred configurations. 
```

Comment:
I am not sure about this one to be honest - what would we like them to do with those APIs regarding an app plugin? Although the local (Grafana instance) APIs can indeed be used to enable / disable an app plugin, I don't think it's something that the plugin developers should be concerned about, it should be handled by the framework (us).

Reply:
If plugin developers don't need to know about the plugins API, then let's delete this reference to it. Making this change.